edc-dataplane/edc-dataplane-base/build.gradle.kts

```diff
@@ -34,7 +34,7 @@ dependencies {
     runtimeOnly(libs.edc.core.did) // for the DID Public Key Resolver
     runtimeOnly(libs.edc.identity.did.web)
     runtimeOnly(libs.edc.config.filesystem)
-    runtimeOnly(libs.edc.auth.tokenbased)
+//    runtimeOnly(libs.edc.auth.tokenbased)
```

Comment:
```suggestion
```

Reply:
yes, I was just discovering that for some reasons I got 401 on the readiness check, commenting that solves the problem, in any case data-plane api are only supposed to be called by control plane so they they don't have to be secured (for the moment), I just removed that dependency because it does not make sense as it is supposed to secure `management-api` endpoints.

Reply:
the Observability API (which contains the readiness check) should not be secured, but it also should never be exposed outside of the pod.

yes, commenting (or better: removing) this line may solve the problem, but if somehow the `TokenBasedAuthenticationService` fires for the Observability API, that would be a bug